streams/writable-streams/aborting.js

```diff
@@ -204,34 +209,64 @@ promise_test(t => {
 }, 'Aborting a WritableStream puts it in an errored state, with a TypeError as the stored error');
 
 promise_test(t => {
-  const ws = new WritableStream();
+  let resolveWritePromise;
```

Comment:
What is the motivation for this change? It seems like a new test; perhaps we can leave the old one alone, and add a second one?